packages/@okta/vuepress-theme-prose/assets/css/components/_content.scss

```diff
@@ -219,6 +219,14 @@
     p > code {
       overflow-wrap: break-word;
     }
+
+    .stack-selector {
+      display: none;
+
+      @include media("<=1439px") {
```

Comment:
Can we use a predefined named size instead of an explicit number here? See below where we use "desktop"